webpack.config.js

```diff
@@ -322,6 +322,13 @@ let appConfig = {
   resolve: {
     alias: {
       app: path.join(staticPrefix, 'app'),
+      '@emotion/styled': path.join(staticPrefix, 'app', 'styled'),
```

Comment:
Oh thoughts on this @EvanPurkhiser? We need to do this to get typing for `styled`

Reply:
Ah I see, seems fine.

Reply:
Maybe drop a comment in here for why we do this